.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,16 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  golangci-lint:
+    name: runner / golangci-lint
```

Comment:
The name is unnecessarily long, rename to `Review`?

Reply:
ok, i will